ts/Series/Map/MapSeries.ts

```diff
@@ -792,7 +790,8 @@ class MapSeries extends ScatterSeries {
             // When dragging or first rendering, animation is off
             } else {
                 transformGroup.attr(merge(
-                    svgTransform, { 'stroke-width': strokeWidth / scale }
+                    svgTransform,
+                    { 'stroke-width': strokeWidth / scale, animator: 1 }
```

Comment:
What does `animator` do? The only place `animator` is in use that can make a difference, is in the [animation that starts on line 776](https://github.com/highcharts/highcharts/blob/6e1d9f13a02172a0a226a698a4884c2b14ad5160/ts/Series/Map/MapSeries.ts#L776-L786). So when we set `animator` to 1, does it mean that that animation is still running, and we're skipping to the end? In that case, the appropriate way of stopping it would be to call the [stop function](https://api.highcharts.com/class-reference/Highcharts#.stop).